src/transforms/filter.js

```diff
@@ -190,85 +187,71 @@ function getFilterFunc(opts, d2c) {
-                    return cv > coercedValue[0] && cv < coercedValue[1];
-                };
-            }
-            else if(coercedStrict[0] && !coercedStrict[1]) {
-                return function(v) {
-                    var cv = d2c(v);
-                    return cv > coercedValue[0] && cv <= coercedValue[1];
-                };
-            }
-            else if(!coercedStrict[0] && coercedStrict[1]) {
-                return function(v) {
-                    var cv = d2c(v);
-                    return cv >= coercedValue[0] && cv < coercedValue[1];
-                };
-            }
-            else if(!coercedStrict[0] && !coercedStrict[1]) {
-                return function(v) {
-                    var cv = d2c(v);
-                    return cv >= coercedValue[0] && cv <= coercedValue[1];
-                };
-            }
-
-            break;
-
-        case 'notwithin':
-
-            if(coercedStrict[0] && coercedStrict[1]) {
-                return function(v) {
-                    var cv = d2c(v);
-                    return cv < coercedValue[0] || cv > coercedValue[1];
-                };
-            }
-            else if(coercedStrict[0] && !coercedStrict[1]) {
-                return function(v) {
-                    var cv = d2c(v);
-                    return cv < coercedValue[0] || cv >= coercedValue[1];
-                };
-            }
-            else if(!coercedStrict[0] && coercedStrict[1]) {
-                return function(v) {
-                    var cv = d2c(v);
-                    return cv <= coercedValue[0] || cv > coercedValue[1];
-                };
-            }
-            else if(!coercedStrict[0] && !coercedStrict[1]) {
-                return function(v) {
-                    var cv = d2c(v);
-                    return cv <= coercedValue[0] || cv >= coercedValue[1];
-                };
-            }
-
-            break;
-
-        case 'in':
+            return function(v) { return d2c(v) > coercedValue; };
+
+        case '>=':
+            return function(v) { return d2c(v) >= coercedValue; };
+
+        case '[]':
+            return function(v) {
+                var cv = d2c(v);
+                return cv >= coercedValue[0] && cv <= coercedValue[1];
+            };
+
+        case '()':
+            return function(v) {
+                var cv = d2c(v);
+                return cv > coercedValue[0] && cv < coercedValue[1];
+            };
+
+        case '[)':
+            return function(v) {
+                var cv = d2c(v);
+                return cv >= coercedValue[0] && cv < coercedValue[1];
+            };
+
+        case '(]':
+            return function(v) {
+                var cv = d2c(v);
+                return cv > coercedValue[0] && cv <= coercedValue[1];
+            };
+
+        case '][':
+            return function(v) {
+                var cv = d2c(v);
+                return cv < coercedValue[0] || cv > coercedValue[1];
```

Comment:
"Some authors use ]a, b[ to denote the complement of the interval (a, b); namely, the set of all real numbers that are either less than or equal to a, or greater than or equal to b." - [wiki](https://en.wikipedia.org/wiki/Interval_(mathematics))

I think it makes intuitive sense also, in that the *retained* part, i.e. the part outside the interval, has _closed_ bounds